python/py-pytest-xdist/Portfile

```diff
@@ -19,15 +19,16 @@ long_description    {*}${description}
 
 homepage            https://github.com/pytest-dev/pytest-xdist
 
-checksums           rmd160  61ce6afcd67bc6ec2c8021cb5710e35a5e22abcc \
-                    sha256  ead156a4db231eec769737f57668ef58a2084a34b2e55c4a8fa20d861107300d \
-                    size    84060
+checksums           rmd160  c9ad106ff930376ecfee16f031c4130f24b13d89 \
+                    sha256  7e578125ec9bc6050861aa93f2d59f1d8d085595d6551c2c90b6f4fad8d3a9f1 \
+                    size    88069
 
-python.versions     37 38 39 310 311 312 313
+python.versions     39 310 311 312 313
```

Comment:
the subports you removed all have dependents; in the future please check this using 
`port echo depends:pyXY-<portname>`